swap_meet/clothing.py

```diff
@@ -0,0 +1,12 @@
+
+from swap_meet.item import Item
+class Clothing(Item):
+    def __init__(self, category = "", condition = 0):
+        self.category = "Clothing"
+        self.condition = condition
```

Comment:
The two lines can be reduced to

```suggestion
        super().__init__("Clothing", condition)
```